Scenario Based Samples/Iframe Full Server Embedding/README.md

```diff
@@ -0,0 +1,76 @@
+# Bold BI Embedded Sample in .NET Core MVC
```

Comment:
Change the Heading suits for this sample.

Reply:
Changed heading